components/server/src/authorization/authorizer.ts

```diff
@@ -120,18 +117,60 @@ export class Authorizer {
         );
     }
 
+    async deleteOrganization(orgID: string): Promise<void> {
+        await this.authorizer.deleteRelationships(
+            v1.DeleteRelationshipsRequest.create({
+                relationshipFilter: v1.RelationshipFilter.create({
+                    resourceType: "organization",
+                    optionalResourceId: orgID,
+                }),
+            }),
+        );
+    }
+
+    async addOrganization(org: Organization, members: TeamMemberInfo[], projects: Project[]): Promise<void> {
+        const updates: v1.RelationshipUpdate[] = [];
```

Comment:
do we need to check the arguments are consistent? i.e. `teamId` in both arrays match the given org.id?

Reply:
Can do an extra sanity check here, yes. I was actually considering reducing the type of `members` to just `{id: <user_id>, role: <role>}` which wouldn't require it

Reply:
Actually, `TeamMemberInfo` doesn't contain the Org ID

Reply:
oh, ok. then we leave it to the caller